content/Solutions/Optimizations/Networking.md

```diff
@@ -0,0 +1,339 @@
+Consistency helps in maintaining predictability and stability in the network environment.
+
+See [Block Shares (iSCSI)]({{< relref "scale/scaletutorials/shares/iscsi/_index.md" >}}) for more information.
+
+#### MPIO for iSCSI
+
+Use MPIO (Multipath I/O) to combine interfaces when using iSCSI.
+Do not use LAGG link aggregation.
+
+MPIO is designed to improve the reliability and performance of data transfers by using multiple physical paths between the iSCSI initiator (client) and the iSCSI target (storage).
+MPIO can distribute I/O traffic across multiple paths, balancing the load and improving overall performance.
+If one path fails, MPIO can automatically switch to an alternate path, ensuring continuous access to the storage.
+
+Both the iSCSI initiator and target must support MPIO for it to be effective.
+MPIO requires configuration on both the initiator and target sides. Each path needs to be properly identified and managed.
+
+To configure multipath for iSCSI, go to the **iSCSI** screen and configure additional Portal IP addresses.
+
+#### iSCSI and VMWare
+
+Refer to the [VMWare best practices guide](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.storage.doc/GUID-BC516B24-7EAB-4ADA-819A-10DC496DEE9B.html) for information on configuring multiple network adapters when using VMWare with your TrueNAS system.
+
+#### Jumbo Frames
+
+Enable jumbo frames on an iSCSI share to enhance storage performance through the transmission of larger data packets, reducing overhead and improving overall efficiency.
+
+To enable jumbo frames, go to the **Network** screen and edit the parent interface(s).
+Set the **MTU** (Maximum Transmission Unit) to 9000.
+Click **SAVE**.
+
+Reboot the system and restart the iSCSI share to inherit.
+
+#### ALUA
+
+{{< enterprise >}}
+TrueNAS Enterprise High Availability (HA) systems should enable ALUA (Asymmetric Logical Unit Access) for iSCSI shares.
+ALUA and MPIO can be used at same time.
+
+ALUA allows a client computer to discover the best path to the storage on a TrueNAS® system. HA storage clusters can provide multiple paths to the same storage. For example, the disks are directly connected to the primary computer and provide high speed and bandwidth when accessed through that primary computer. The same disks are also available through the secondary computer, but because they are not directly connected to it, speed and bandwidth are restricted. With ALUA, clients automatically ask for and use the best path to the storage. If one of the TrueNAS® HA computers becomes inaccessible, the clients automatically switch to the next best alternate path to the storage. When a better path becomes available, as when the primary host becomes available again, the clients automatically switch back to that better path to the storage.
+
+To enable ALUA, select **Enable iSCSI ALUA** from the **Target Global Configuration** tab on the **iSCSI** screen.
+{{< /enterprise >}}
+
+## Multicast
+
+{{< enterprise >}}
```

Comment:
Truthfully I don't know if this section is worthwhile. VRRP is used on SCALE ent rather than CARP, and hopefully our deployment teams would catch anything like this during deployment.